src/test/java/com/wikia/webdriver/elements/mercury/components/Navigation.java

```diff
@@ -173,6 +187,15 @@ public boolean isLogoutLinkVisible() {
     }
   }
 
+  public boolean isExploreWikiHeaderVisible() {
+    try {
+      return exploreWikiHeader.isDisplayed();
```

Comment:
AFAIR this logic is duplicated in some other places. Did you consider to extract it somewhere and pass the element as an argument?